pyproject.toml

```diff
@@ -85,6 +85,7 @@ docs = [
     'pydata-sphinx-theme>=0.15.4',
     'PyWavelets>=1.1.1',
     'pytest-doctestplus',
+    'joblib',
```

Comment:
Should dependency be `sphinx-gallery[parallel]` instead?

Reply:
Yes, that's the better solution! Addressed in https://github.com/scikit-image/scikit-image/pull/7579/commits/09b24f5ccefc0e88d392c685d8e33a8f9776fa29.